packages/flutter_tools/test/commands.shard/hermetic/devices_test.dart

```diff
@@ -34,7 +34,7 @@ void main() {
       expect(testLogger.statusText, containsIgnoringWhitespace('No devices detected'));
     }, overrides: <Type, Generator>{
       AndroidSdk: () => null,
-      DeviceManager: () => DeviceManager(),
+      DeviceManager: () => NoDevicesManager(),
```

Comment:
This fails with devices attached!

Reply:
Looking at the implementation of `NoDevicesManager`...should we just delete this test?

Reply:
I'm sort of assuming this is testing the devices command logic for error states and not the devicemanager. I think `AndroidSdk: () => null,` was an early attempt to disable all devices, but that doesn't work in a web/desktop world

Reply:
Well, I guess we're still testing some code here. I take it back.